src/Block/TreeBlockService.php

```diff
@@ -14,26 +14,21 @@
 namespace Sonata\DoctrinePHPCRAdminBundle\Block;
 
 use Sonata\AdminBundle\Form\FormMapper;
-use Sonata\BlockBundle\Block\BaseBlockService;
 use Sonata\BlockBundle\Block\BlockContextInterface;
+use Sonata\BlockBundle\Block\Service\AbstractBlockService;
 use Sonata\BlockBundle\Model\BlockInterface;
 use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\OptionsResolver\Options;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class TreeBlockService extends BaseBlockService
+class TreeBlockService extends AbstractBlockService
 {
     /**
      * @var array
      */
     protected $defaults;
 
-    /**
-     * @param string          $name
```

Comment:
this line should be kept, until we can use a typehint `string` in the parameter signature